### PR TITLE
Adds additional movie titles to Bay's End of Round Credits.

### PR DESCRIPTION
**Pull request by Yantarii**
Adds a whole bunch of additional movie titles to the round end, even if we baymerge again, it shouldn't be too hard to just copy our new one.

Removed references to GAS and Serpentids, and added IPCs and Resomi.

### CODE REVIEW

File: code/game/movietitles.dm [hunk 1]
```diff
@@ -98,16 +98,17 @@ GLOBAL_LIST(end_titles)
...
+		possible_titles += "THE CREW LEARNS ABOUT [pick("LOVE", "DRUGS", "THE DANGERS OF MONEY LAUNDERING", "THE FASHION POLICE", "XENIC SENSITIVITY", "PIRATES, PIRATES, PIRATES!", "DISPOSAL CHUTES", "SUIT SENSORS", "GET RICH QUICK SCHEMES", "HEALTH INSURANCE", "INVESTMENT FRAUD", "KELOTANE ABUSE", "HYPERZINE SNORTING", "COMBINING WATER AND POTASSIUM", "RADIATION PROTECTION", "SACRED GEOMETRY", "STRING THEORY", "ABSTRACT MATHEMATICS",\
+							"[pick("UNATHI", "SKRELLIAN", "DIONAN", "RESOMI", "VOX", "IPC")] MATING RITUALS", "ANCIENT CHINESE MEDICINE", "THE IMPORTANCE OF [pick("AIRLOCK SAFETY", "MAINTENANCE", "SUIT SENSORS", "FLOOR PILLS", "ELEVATORS", "HEIGHTS", "TORPEDOS", "GOOD RELATIONS", "A FULLY STOCKED BAR", "AWAY MISSIONS", "MANDATORY DRUG TESTING", "HIRING SECURITY", "GOOD FORENSICS", "TEST WIRES", "UNIONS", "NEURAL LACES", "ENGINEERS")]")]"
+		possible_titles += "A VERY [pick("CORPORATE", "NANOTRASEN", "FLEET", "TERRAN", "LACTERAN", "HAPHAESTUS", "DAIS", "XENOLIFE", "EXPEDITIONARY", "DIONAN", "PHORON", "MARTIAN", "RESOMI")] [pick("CHRISTMAS", "EASTER", "HOLIDAY", "WEEKEND", "THURSDAY", "VACATION", "FRIDAY", "MONDAY", "NEW YEARS", "BOXING DAY")]"
```
Comment (CPU-Blanc) [inline code/game/movietitles.dm:107]: Could be fun to use the actual day here instead :P
```suggestion
		possible_titles += "A VERY [pick("CORPORATE", "NANOTRASEN", "FLEET", "TERRAN", "LACTERAN", "HAPHAESTUS", "DAIS", "XENOLIFE", "EXPEDITIONARY", "DIONAN", "PHORON", "MARTIAN", "RESOMI")] [pick("CHRISTMAS", "EASTER", "HOLIDAY", "VACATION", uppertext(time2text(world.realtime, "Day")), "NEW YEARS", "BOXING DAY")]"
```

File: code/game/movietitles.dm [hunk 2]
```diff
@@ -98,16 +98,17 @@ GLOBAL_LIST(end_titles)
...
-		possible_titles += "[pick("GUNS, GUNS EVERYWHERE", "THE LITTLEST ARMALIS", "WHAT HAPPENS WHEN YOU MIX MAINTENANCE DRONES AND COMMERCIAL-GRADE PACKING FOAM", "ATTACK! ATTACK! ATTACK!", "SEX BOMB", "THE LEGEND OF THE ALIEN ARTIFACT: PART [pick("I","II","III","IV","V","VI","VII","VIII","IX", "X", "C","M","L")]")]"
+		possible_titles += "THE [pick("DOWNFALL OF ", "RISE OF ", "TROUBLE WITH ", "FINAL STAND OF ", "DARK SIDE OF ", "THE LACK OF ", "DESOLATION OF ", "DESTRUCTION OF ", "THE ISSUE WITH ", "CRISIS OF ")]\
+							[pick("SPACEMEN", "HUMANITY", "DIGNITY", "SANITY", "THE CHIMPANZEES", "THE VENDOMAT PRICES", "CARGONIA", "BLUESPACE", "THE ARTIFICAL INTELLIGENCE UNIT", "THE MIME", "THE CLOWN (AGAIN)", "THE CHAPLAIN", "THE BALDIE", "IAN", "THE EXOPLANET",\
+							"THE SUPERMATTER CRYSTAL", "MEDICAL", "ENGINEERING", "SECURITY", "RESEARCH", "THAT ONE LAST CREWMATE", "THE SERVICE DEPARTMENT", "COMMAND", "THE EXPLORERS", "THE AWAY TEAM", "THE BOTANIST", "THE CHEF", "BREATHABLE AIR", "THE KITCHEN", "THE BAR", "THE BARTENDER", "[uppertext(GLOB.using_map.station_name)]")]"
```
Comment (CPU-Blanc) [inline code/game/movietitles.dm:103]:
```suggestion
							"THE SUPERMATTER CRYSTAL", "MEDICAL", "ENGINEERING", "SECURITY", "RESEARCH", "THAT ONE LAST CREWMATE", "THE SERVICE DEPARTMENT", "COMMAND", "THE EXPLORERS", "THE AWAY TEAM", "THE BOTANIST", "THE CHEF", "BREATHABLE AIR", "THE KITCHEN", "THE BAR", "THE BARTENDER", "THE [uppertext(GLOB.using_map.station_name)]")]"
```

File: code/game/movietitles.dm [hunk 3]
```diff
@@ -98,16 +98,17 @@ GLOBAL_LIST(end_titles)
...
 		possible_titles += "[pick("SPACE", "SEXY", "DRAGON", "WARLOCK", "LAUNDRY", "GUN", "ADVERTISING", "DOG", "CARBON MONOXIDE", "NINJA", "WIZARD", "SOCRATIC", "JUVENILE DELIQUENCY", "POLITICALLY MOTIVATED", "RADTACULAR SICKNASTY")] [pick("QUEST", "FORCE", "ADVENTURE")]"
 		possible_titles += "[pick("THE DAY [uppertext(GLOB.using_map.station_short)] STOOD STILL", "HUNT FOR THE GREEN WEENIE", "ALIEN VS VENDOMAT", "SPACE TRACK")]"
+		possible_titles += "THE CREW [pick("FALL OUT A AIRLOCK", "LOOKS UP", "FUCKING DIES", "DESTROY THE [uppertext(GLOB.using_map.station_short)]", "DON'T DO ANYTHING", "GOES TO HELL", "GOES TO SPACE PRISON", "EATS PACKING PEANUTS", "FORGET HOW TO BE KIND",  "MOVE TO A EXOPLANET", "CATCH A STOWAWAY", "FALL OUT A WINDOW", "SETTLE A LAWSUIT", "BUY A NEW SHIP", "FIND A DEAD GUY", "LOVES [pick("MEDBAY", "ALCOHOL", "YOU", "UNREGISTERED GUNS", "MONKEY CUBES", "MAKING A MESS", "ANNOYING COMMAND", "GETTING BRIGGED")]")]"
```
Comment (CPU-Blanc) [inline code/game/movietitles.dm:111]:
```suggestion
		possible_titles += "THE CREW [pick("FALL OUT AN AIRLOCK", "LOOKS UP", "FUCKING DIES", "DESTROY THE [uppertext(GLOB.using_map.station_short)]", "DON'T DO ANYTHING", "GOES TO HELL", "GOES TO SPACE PRISON", "EATS PACKING PEANUTS", "FORGET HOW TO BE KIND",  "MOVE TO AN EXOPLANET", "CATCH A STOWAWAY", "FALL OUT A WINDOW", "SETTLE A LAWSUIT", "BUY A NEW SHIP", "FIND A DEAD GUY", "LOVES [pick("MEDBAY", "ALCOHOL", "YOU", "UNREGISTERED GUNS", "MONKEY CUBES", "MAKING A MESS", "ANNOYING COMMAND", "GETTING BRIGGED")]")]"
```

File: code/game/movietitles.dm [hunk 4]
```diff
@@ -98,16 +98,17 @@ GLOBAL_LIST(end_titles)
...
+		possible_titles += "THE CREW LEARNS ABOUT [pick("LOVE", "DRUGS", "THE DANGERS OF MONEY LAUNDERING", "THE FASHION POLICE", "XENIC SENSITIVITY", "PIRATES, PIRATES, PIRATES!", "DISPOSAL CHUTES", "SUIT SENSORS", "GET RICH QUICK SCHEMES", "HEALTH INSURANCE", "INVESTMENT FRAUD", "KELOTANE ABUSE", "HYPERZINE SNORTING", "COMBINING WATER AND POTASSIUM", "RADIATION PROTECTION", "SACRED GEOMETRY", "STRING THEORY", "ABSTRACT MATHEMATICS",\
+							"[pick("UNATHI", "SKRELLIAN", "DIONAN", "RESOMI", "VOX", "IPC")] MATING RITUALS", "ANCIENT CHINESE MEDICINE", "THE IMPORTANCE OF [pick("AIRLOCK SAFETY", "MAINTENANCE", "SUIT SENSORS", "FLOOR PILLS", "ELEVATORS", "HEIGHTS", "TORPEDOS", "GOOD RELATIONS", "A FULLY STOCKED BAR", "AWAY MISSIONS", "MANDATORY DRUG TESTING", "HIRING SECURITY", "GOOD FORENSICS", "TEST WIRES", "UNIONS", "NEURAL LACES", "ENGINEERS")]")]"
```
Comment (CPU-Blanc) [inline code/game/movietitles.dm:106]:
```suggestion
							"[pick("UNATHI", "SKRELLIAN", "DIONAN", "RESOMI", "VOX", "IPC")] MATING RITUALS", "ANCIENT CHINESE MEDICINE", "THE IMPORTANCE OF [pick("AIRLOCK SAFETY", "MAINTENANCE", "SUIT SENSORS", "FLOOR PILLS", "ELEVATORS", "HEIGHTS", "TORPEDOES", "GOOD RELATIONS", "A FULLY STOCKED BAR", "AWAY MISSIONS", "MANDATORY DRUG TESTING", "HIRING SECURITY", "GOOD FORENSICS", "TEST WIRES", "UNIONS", "NEURAL LACES", "ENGINEERS")]")]"
```

File: code/game/movietitles.dm [hunk 5]
```diff
@@ -98,16 +98,17 @@ GLOBAL_LIST(end_titles)
...
+		possible_titles += "A VERY [pick("CORPORATE", "NANOTRASEN", "FLEET", "TERRAN", "LACTERAN", "HAPHAESTUS", "DAIS", "XENOLIFE", "EXPEDITIONARY", "DIONAN", "PHORON", "MARTIAN", "RESOMI")] [pick("CHRISTMAS", "EASTER", "HOLIDAY", "WEEKEND", "THURSDAY", "VACATION", "FRIDAY", "MONDAY", "NEW YEARS", "BOXING DAY")]"
+		possible_titles += "[pick("GUNS, GUNS EVERYWHERE", "THE LITTLEST ARMALIS", "WHAT HAPPENS WHEN YOU MIX MAINTENANCE DRONES AND COMMERCIAL-GRADE PACKING FOAM", "INTERNALLY COHERANT, HARDCORE CRAZY TO THE MEGA", "ATTACK! ATTACK! ATTACK!", "SEX BOMB", "THE LEGEND OF THE ALIEN ARTIFACT: PART [pick("I","II","III","IV","V","VI","VII","VIII","IX", "X", "C","M","L")]")]"
```
Comment (CPU-Blanc) [inline code/game/movietitles.dm:108]:
```suggestion
		possible_titles += "[pick("GUNS, GUNS EVERYWHERE", "THE LITTLEST ARMALIS", "WHAT HAPPENS WHEN YOU MIX MAINTENANCE DRONES AND COMMERCIAL-GRADE PACKING FOAM", "INTERNALLY COHERENT, HARDCORE CRAZY TO THE MEGA", "ATTACK! ATTACK! ATTACK!", "SEX BOMB", "THE LEGEND OF THE ALIEN ARTIFACT: PART [pick("I","II","III","IV","V","VI","VII","VIII","IX", "X", "C","M","L")]")]"
```

File: code/game/movietitles.dm [hunk 6]
```diff
@@ -98,16 +98,17 @@ GLOBAL_LIST(end_titles)
...
 		possible_titles += "[pick("SPACE", "SEXY", "DRAGON", "WARLOCK", "LAUNDRY", "GUN", "ADVERTISING", "DOG", "CARBON MONOXIDE", "NINJA", "WIZARD", "SOCRATIC", "JUVENILE DELIQUENCY", "POLITICALLY MOTIVATED", "RADTACULAR SICKNASTY")] [pick("QUEST", "FORCE", "ADVENTURE")]"
```
Comment (CPU-Blanc) [inline code/game/movietitles.dm:109]: I know this wasn't you, but.. mistake here lmao
```suggestion
		possible_titles += "[pick("SPACE", "SEXY", "DRAGON", "WARLOCK", "LAUNDRY", "GUN", "ADVERTISING", "DOG", "CARBON MONOXIDE", "NINJA", "WIZARD", "SOCRATIC", "JUVENILE DELINQUENCY", "POLITICALLY MOTIVATED", "RADTACULAR SICKNASTY")] [pick("QUEST", "FORCE", "ADVENTURE")]"
```

**Review by CPU-Blanc — CHANGES_REQUESTED** [review]
Some spelling and grammar issues

**Comment by Yantarii** [comment]
Should be most of it fixed!

**Review by CPU-Blanc — APPROVED** [review]
LGTM

**Pull request merged**